src/nucleic_acid/nucleic_acid.jl

```diff
@@ -5,26 +5,16 @@ const _πT(mod) = _πACGT(mod)
 const _πACGU(mod) = _πACGT(mod)
 
 
-@inline function _πACGT(mod::NASM)
-  return 0.25
-end
-
-@inline function _π(mod::NASM)
-  return SVector(_πA(mod), _πC(mod), _πG(mod), _πT(mod))
-end
-
+_πACGT(mod::NASM) = 0.25
 
-@inline function _πR(mod::NASM)
-  return _πA(mod) + _πG(mod)
-end
+_π(mod::NASM) = SVector(_πA(mod), _πC(mod), _πG(mod), _πT(mod))
```

Comment:
Removed @inline from these functions as they are so simple, and single line definitions, and so the julia compiler will in all likelihood inline them anyway. @inline is typically required for functions where you want to try to get the compiler to inline small functions which a a bit bigger than one liners, especially when you know they will be called a lot, especially in loops. So the iteration methods start, next, done, and a few others like indexing methods are good candidates for the @inline macro. But one liner functions, with a few operations in them, can be assumed to be inlined.

Reply:
Okay thanks!